__std__/flake.nix

```diff
@@ -16,17 +16,23 @@
   # { inputs, cell }: inputs.nixpkgs, inputs.haskell-nix
   inputs = {
     nixpkgs = {
-      url = "github:NixOS/nixpkgs/34e4df55664c24df350f59adba8c7a042dece61e";
+      url = "github:NixOS/nixpkgs";
     };
     std = {
       url = "github:divnix/std";
       inputs.nixpkgs.follows = "nixpkgs";
     };
     haskell-nix = {
-      url = "github:input-output-hk/haskell.nix/e36a34c4abc02287ec79bf88a0c993f6b66f9945";
+      url = "github:input-output-hk/haskell.nix";
+      inputs = {
+        hackage.follows = "hackage-nix";
+        nixpkgs.follows = "nixpkgs";
+        # Ignore stackage (self reference)
```

Comment:
Can you say a bit more? Or can we just leave out this line?

Reply:
this was an "hack" to avoid downloading new stackage each time we update haskell.nix (since we don't use stackage).
But can be removed.

Reply:
But not sure why that was necessary, since inputs loading should lazy now. I'll remove for now.